ironfish-rust-nodejs/index.js

```diff
@@ -275,6 +275,10 @@ module.exports.DECRYPTED_NOTE_LENGTH = DECRYPTED_NOTE_LENGTH
 module.exports.Note = Note
 module.exports.TransactionPosted = TransactionPosted
 module.exports.PROOF_LENGTH = PROOF_LENGTH
+module.exports.TRANSACTION_SIGNATURE_LENGTH = TRANSACTION_SIGNATURE_LENGTH
+module.exports.TRANSACTION_PUBLIC_KEY_LENGTH = TRANSACTION_PUBLIC_KEY_LENGTH
+module.exports.TRANSACTION_EXPIRATION_LENGTH = TRANSACTION_EXPIRATION_LENGTH
```

Comment:
Should this just be something `SEQUENCE_LENGTH`?
```suggestion
module.exports.SEQUENCE_LENGTH = SEQUENCE_LENGTH
```

Reply:
I want to keep the original name since its referring to the transaction expiration length. I know the type of expiration is a sequence.  If we want to change the type of expiration to other class X, then we can not just update this constant value, instead we need to change all the `SEQUENCE_LENGTH` constants to a new X_LENGTH.